qa/setup_dali_extra.sh

```diff
@@ -2,11 +2,12 @@
 
 # Fetch test data
 export DALI_EXTRA_PATH=${DALI_EXTRA_PATH:-/opt/dali_extra}
-export DALI_EXTRA_URL=${DALI_EXTRA_URL:-"https://github.com/NVIDIA/DALI_extra.git"}
+export DALI_EXTRA_URL="https://github.com/jantonguirao/DALI_extra.git"
 
 DIR=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )
-DALI_EXTRA_VERSION_PATH="${DIR}/../DALI_EXTRA_VERSION"
-read -r DALI_EXTRA_VERSION < ${DALI_EXTRA_VERSION_PATH}
+#DALI_EXTRA_VERSION_PATH="${DIR}/../DALI_EXTRA_VERSION"
+#read -r DALI_EXTRA_VERSION < ${DALI_EXTRA_VERSION_PATH}
```

Comment:
Why?

Reply:
Testing with a new image that I added to DALI_extra, I'm going to remove it before merging